etc/config/c++.amazon.properties

```diff
@@ -466,6 +466,10 @@ compiler.clang_bb_p2996.exe=/opt/compiler-explorer/clang-bb-p2996-trunk/bin/clan
 compiler.clang_bb_p2996.semver=(experimental P2996)
 compiler.clang_bb_p2996.notification=Experimental Reflection Support; see <a href="https://github.com/bloomberg/clang-p2996/blob/p2996/P2996.md" target="_blank" rel="noopener noreferrer">P2996<sup><small class="fas fa-external-link-alt opens-new-window" title="Opens in a new window"></small></sup></a>
 compiler.clang_bb_p2996.options=-std=c++26 -freflection -stdlib=libc++
+compiler.clang_p2998.exe=/opt/compiler-explorer/clang-p2998-trunk/bin/clang++
+compiler.clang_p2998.semver=(experimental P2998)
+compiler.clang_p2998.notification=Experimental support for additional template argument deduction from a function call; see <a href="https://github.com/Bekenn/llvm-project/tree/p2998" target="_blank" rel="noopener noreferrer">P2998<sup><small class="fas fa-external-link-alt opens-new-window" title="Opens in a new window"></small></sup></a>
+compiler.clang_p2998.options=-p2998
```

Comment:
does this maybe also require libc++? if so it would be best to add -stdlib=libc++

Reply:
There's no library component for the feature, but I have no objection to adding that.  Is that typical for clang compilers?

Reply:
No no, not at all, I just didn't read what the proposal was about, saw the word template and figured I'd ask. All good like this

Reply:
(Although it might make sense to update some deduction guides from the standard... I'll consider that for future editions.)